template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -210,15 +223,18 @@ def check_uid_uniqueness(proof_dir, proof_uids):
     sys.exit(1)
 
 
-async def configure_proof_dirs(queue, counter, proof_uids):
+async def configure_proof_dirs(queue, counter, proof_uids, enable_pools):
     while True:
         print_counter(counter)
         path = str(await queue.get())
 
         check_uid_uniqueness(path, proof_uids)
 
+        pools = ["ENABLE_POOLS=true"] if enable_pools else []
+
         proc = await asyncio.create_subprocess_exec(
-            "nice", "-n", "15", "make", "-B", "--quiet", "_report", cwd=path)
+            "nice", "-n", "15", "make", *pools, "-B", "--quiet", "_report",
```

Comment:
`nice` is a Unix command that allows you to change the CPU preemption level of a process, where 20 means that the wrapped process can be starved of CPU resources by any other process, and -20 means that the process is run at highest priority and can interrupt other processes. See `man nice`.

I'm using it here so that the proofs can be configured without interrupting real-time tasks like having a video call or rendering the user interface. We're spawning a lot of processes in parallel here.

Reply:
(Incidentally, using it in the starter kit is probably a good idea too, doing `nice -n 15 cbmc`. I doubt it would increase the runtime by very much, but it would make running the proofs on a laptop more tolerable. Not sure if people actually run all the proofs on their laptop at once, though.)

Reply:
Nice! Shall we add this explanation as comment there? @karkhaz

Reply:
Done